sympy/solvers/tests/test_solvers.py

```diff
@@ -1624,8 +1624,8 @@ def test_rewrite_trig():
     assert solve(sinh(x) + tanh(x)) == [0, I*pi]
 
     # issue 6157
-    assert solve(2*sin(x) - cos(x), x) == [-2*atan(2 + sqrt(5)),
-                                           -2*atan(-sqrt(5) + 2)]
+    assert solve(2*sin(x) - cos(x), x) == [-2*atan(2 - sqrt(5)),
+                                           -2*atan(2 + sqrt(5))]
```

Comment:
This one is the reverse of the above. We had a+b,-b+a and now we have a-b,a+b which is nicer.